src/scss/components/_banner.scss

```diff
@@ -42,7 +47,7 @@
   }
 
   &,
-  &[aria-expanded=true] {
+  &[aria-expanded='true'] {
```

Comment:
this is a style thing right? there aren't "types" in CSS selectors where this would matter?

Reply:
> this is a style thing right? there aren't "types" in CSS selectors where this would matter?

Right, just styling. Prompted for consistency's sake due to these intending to explicitly override selector styles from USWDS, which include quotes (though USWDS uses double quotes).

https://github.com/uswds/uswds/blob/2650f15/src/stylesheets/components/_banner.scss#L251